openedx/core/djangoapps/content_libraries/tests/test_runtime.py

```diff
@@ -181,10 +182,29 @@ def test_xblock_metadata(self):
 
 
 @requires_blockstore
+# EphemeralKeyValueStore requires a working cache, and the default test cache doesn't work:
+@override_settings(XBLOCK_RUNTIME_V2_EPHEMERAL_DATA_CACHE='blockstore')
```

Comment:
I'm not sure what the reasoning for using `default` as the default cache here is..  You added the `CACHES['blockstore']` setting as part of https://github.com/openedx/edx-platform/pull/21630, before you added this setting as part of https://github.com/openedx/edx-platform/pull/22427.

The only reason I can think of is if people are overriding `settings.CACHES`, then maybe they don't have a `blockstore` entry in their custom configs yet, and so using `blockstore` as the default here would break things?

I'd prefer not to change that here if we don't have to, but will move this setting to `cms.envs.test` so we don't have to override it here.